chimney/src/test/scala/io/scalaland/chimney/TotalTransformerJavaBeansSpec.scala

```diff
@@ -44,6 +44,23 @@ class TotalTransformerJavaBeansSpec extends ChimneySpec {
     }
   }
 
+  group("""settings .withField*(_.getTo, ...) and .withFieldRenamed(_.from, _.getTo)""") {
+
+    test("transform case class to Java Bean, allowing using getters as a way to rename into matching setters") {
+      val source = CaseClassWithFlagRenamed("test-id", "test-name", renamedFlag = true)
+      val target = source
+        .into[JavaBeanTarget]
+        .withFieldConst(_.getId, source.id)
```

Comment:
How about adding a case when getter is not paired with setter with the same name? We should expect compilation to fail.

Reply:
Added test, it required some change because code without it could successfully compile if ignored field override would make it compile.